datamade/how-to
# Port search documentation to its own directory

## Overview

This PR transitions our draft of search best practices to a directory in this repo, closes #4.

## Testing Instructions

- View the diff on GitHub
- Confirm the markdown looks good
